recipes/tests.rb

```diff
@@ -638,3 +638,24 @@ module load armpl/#{node['cfncluster']['armpl']['version']}
     user node['cfncluster']['cfn_cluster_user']
   end
 end
+
+##################
+# Verify enough space on AMIs
+###################
+unless node['cfncluster']['os'].end_with?("-custom")
+  bash 'verify 10 GB of space left on root volume' do
+    cwd Chef::Config[:file_cache_path]
+    # This test assumes the df output is as follows:
+    # $ df --block-size GB /
+    # Filesystem     1GB-blocks  Used Available Use% Mounted on
+    # overlay              63GB  18GB      42GB  30% /
+    code <<-CAPACITY_CHECK
+      free_gigs="$(df --block-size GB / | tail -n1 | awk '{print $4}' | cut -d G -f1)"
```

Comment:
You might consider using `--output=iavail` to just get the available from the `df` command.